src/simmate/apps/badelf/models/badelf.py

```diff
@@ -15,6 +15,13 @@ class BadElf(Structure, Calculation):
     class Meta:
         app_label = "workflows"
```

Comment:
I missed this in your last PR but this `Meta` tag and the app label need to be removed until badelf becomes a default app. Also the migrations in this MR need to be deleted and recreated (and they'll then show up in the badelf app).

All this will do is change your table name from `workflows_badelf` to `badelf_badelf` in the postgres database -- but organizing the workflow migrations is the bigger deal here and what would cause trouble for users + db maintainers.

Reply:
Ok cool. I'm making a clean simmate dev install and remaking the migrations.

Reply:
if you want the table to still show as `workflows_badelf` in the postgres db: https://docs.djangoproject.com/en/5.0/ref/models/options/#table-names

``` python
class Example(models.Model):
    . . .
    class Meta:
        db_table = "workflows_badelf"
```